### ISSUE TITLE
How to get the error code?

### ISSUE THREAD

**Issue by merbin2012** [admob-plus/admob-plus#63]
I have decided to use multiple ads network, but when I don't have fill rate (error code 3), I will request another network. So I need to know the error code on client side. But I can get it in java code.

**I am can not able to find the error code in data argument**

```
document.addEventListener('admob.banner.load_fail', (data) => {
  console.log("Banner Load Fail");
  debugger;  
})
```

**Comment by DoruLaz** [comment]
For AdMob I found the error code here: **error.detail.errorCode**, but I think the error code is dependent on the ad network used. Maybe you should debug for all ad networks you try to use..

**Comment by merbin2012** [comment]
@DoruLaz Hi, thanks for your replay.  Above step (error.detail.errorCode) is not working, please check the below-attached screenshot.

![errorrr](https://user-images.githubusercontent.com/3687184/50959997-573c8e80-14ea-11e9-81ed-e4986e7646b3.png)

**Comment by DoruLaz** [comment]
What I found was that the error method does not always contain error code. Sometimes I get an error object that is only like this:

`{
    istrusted: false
}
`

But when you have the NoFill (no ads available) error from AdMob, you will usually have error code 3 defined in **error.detail.errorCode**.

I use it like this:

`
admob.interstitial.load({
    id: {
        android: ENV.adMob.androidInterstitialId,
        ios: ENV.adMob.iOSInterstitialId
    },
}).then(
    function () {
        admob.interstitial.show().then(
            function () {
                // Success
            },
            function (error) {
                // Error on show. 
                console.log(error);
            }
        );
    },
    function (error) {        
        if (error.detail && error.detail.errorCode && error.detail.errorCode === 3) {
            // No ad from admob. This is not actually an error,
            // AdMob does not have ads for me at this moment.
            // You could see if you have ads from another ad provider here if you only want to do that on no fill.
        }
        else {
            // Another error.
            console.log(error);
        }
    }
);
`

Basically if you get the error function called, you may try to get an ad from another network there, if that's what you want.
